prism.el

```diff
@@ -901,34 +905,35 @@ arguments to set the same faces."
              (symbol-function 'ignore)))
     ;; Avoid saving the file for each variable, which is very slow.
     ;; Save it once at the end.
-    (dolist (var (list 'prism-desaturations 'prism-lightens 'prism-num-faces
-                       'prism-comments-fn 'prism-strings-fn))
+    (dolist (var (list 'prism-desaturations 'prism-lightens 'prism-opacities 'prism-num-faces
+                       'prism-comments-fn 'prism-strings-fn 'prism-parens-fn))
```

Comment:
> Let me begin by asking this, before I forget: Have you signed the FSF copyright assignment? Since I now have a package in GNU ELPA, I'm thinking about contributing some more of mine to it, and this one seems like an obvious candidate. This would be a non-trivial change according to FSF guidelines, so it would require CA if it were in ELPA. (I apologize for not raising this issue sooner, but this is a recent development for me, and I'm still considering how it applies to my packages and to which ones.)

I have not yet signed the copyright assignment, but I'll look into it and get back to you.